src/core_plugins/kibana/server/tutorials/netflow/index.js

```diff
@@ -0,0 +1,65 @@
+import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
+import { INSTRUCTION_VARIANT } from '../../../common/tutorials/instruction_variant';
+
+export function netflowSpecProvider() {
+  return {
+    id: 'netflow',
+    name: 'Netflow',
+    category: TUTORIAL_CATEGORY.SECURITY,
+    shortDescription: 'Collect Netflow records sent by a Netflow exporter',
+    longDescription: 'The Logstash Netflow module simplifies the collection, normalization, and visualization of network flow data. ' +
+      'With a single command, the module parses network flow data, indexes the events into Elasticsearch, and installs a suite of Kibana ' +
+      'dashboards to get you exploring your data immediately. Logstash modules support Netflow Version 5 and 9. [Learn more]' +
+      '({config.elastic_docs.website_url}/guide/en/logstash/{config.elastic_docs.link_version}/netflow-module.html) about the Netflow ' +
+      'module.',
+    //iconPath: '', TODO
+    completionTimeMinutes: 10,
+    //previewImagePath: 'kibana-apache.png', TODO
+    params: [
+      {
+        'netflow.var.input.udp.port': {
+          type: 'number', // TODO: Make this a const as well?
+          defaultValue: 2055
+        }
+      }
+    ],
+    instructionSets: [
+      {
+        title: 'Getting Started',
+        instructionVariants: [
+          {
+            id: INSTRUCTION_VARIANT.OSX,
+            instructions: [
+              {
+                title: 'Download and install Logstash',
+                textPre: 'Skip this step if Logstash is already installed. First time using Logstash? See the ' +
+                         '[Getting Started Guide]({config.elastic_docs.website_url}/guide/en/logstash/{config.elastic_docs.link_version}' +
+                         '/getting-started-with-logstash.html).',
+                commands: [
+                  'curl -L -O https://artifacts.elastic.co/downloads/logstash/logstash-{config.kibana.version}.tar.gz',
+                  'tar xzvf logstash-{config.kibana.version}.tar.gz'
+                ]
+              },
+              {
+                title: 'Setup the Netflow module',
+                textPre: 'In the Logstash install directory, run the following command to setup the Netflow module.',
+                commands: [
+                  './bin/logstash --modules netflow --setup',
```

Comment:
Are there any platforms where the leading `./` actually required to run Logstash? Not sure, but I'm able to run Logstash on mac with `bin/logstash`, and that's what we use all over the Logstash docs. TBH, I wish Beats could do the same.